libs/sdk-core/src/models.rs

```diff
@@ -142,12 +142,38 @@ pub struct ReverseSwapPairInfo {
     pub fees_hash: String,
     /// Percentage fee for the reverse swap service
     pub fees_percentage: f64,
-    /// Estimated miner fees in sats for locking up funds
+    /// Estimated miner fees in sats for locking up funds, assuming a transaction virtual size of
+    /// [`crate::ESTIMATED_LOCKUP_TX_VSIZE`] vbytes and [ReverseSwapPairInfo::feerate]
     pub fees_lockup: u64,
-    /// Estimated miner fees in sats for claiming funds
+    /// Estimated miner fees in sats for claiming funds, assuming a transaction virtual size of
+    /// [`crate::ESTIMATED_CLAIM_TX_VSIZE`] vbytes and [ReverseSwapPairInfo::feerate]
     pub fees_claim: u64,
 }
 
+impl ReverseSwapPairInfo {
+    /// Estimates the net amount received at the destination onchain address, for these fees and a given
+    /// `send_amount_sat`
+    pub fn estimate_amount_sat_received(&self, send_amount_sat: u64) -> Result<u64> {
```

Comment:
How would we expose that to the bindings (uniffi/dart/react native) ?

Reply:
I replaced the `impl ReverseSwapPairInfo` with an extra optional arg in the service method `fetch_reverse_swap_fees`.